### DISCUSSION TITLE
Welcome to desafio-github-markdown Discussions!

### DISCUSSION THREAD

**Discussion by joao326** [joao326/desafio-github-markdown#2] [Announcements]
Teste de discussões

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jotaphe** [comment]
Olá. Sou a conta 2 do João :)

**Reply by joao326** [reply]
Prazer! Sou a conta 1 :)

**Comment by jotaphe** [comment]
# Próximos passos
1. Criar duas branches: uma para resumir tudo que aprendi de maneira simples e outra para subir todos os conteúdos anotados.

**Reply by joao326** [reply]
Show! Vou criar
- Branch1: resumo
- Branch2: adiçãoDeConteúdos
